#
# Patch to allow the t_kdb.py test to test a KDC using a LDAP backend provided
# by the Solaris OpenLDAP slapd for the KDB.  MIT have been notified and they
# say they will update t_kdb.py to work on S12 by the 1.15 release.
#
# Update May 2017:
# Majority of the patch is no longer needed, replaced by setting new env.
# var SLAPD. All we now need to do is set mdb db backend.
#
# Patch source: in-house
#

--- a/src/tests/t_kdb.py
+++ b/src/tests/t_kdb.py
@@ -60,8 +60,7 @@ file.write('pidfile %s\n' % slapd_pidfile)
 file.write('include %s\n' % schema)
 if core_schema:
     file.write('include %s\n' % core_schema)
-file.write('moduleload back_bdb\n')
-file.write('database bdb\n')
+file.write('database mdb\n')
 file.write('suffix %s\n' % top_dn)
 file.write('rootdn %s\n' % admin_dn)
 file.write('rootpw %s\n' % admin_pw)
